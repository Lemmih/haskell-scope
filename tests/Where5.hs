module Where5 where

-- Similar to Where2 but with a function declaration instead of a pattern.
a x = b
  where b = b

a' x = b
