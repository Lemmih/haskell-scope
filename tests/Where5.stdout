Scope errors:
  Value not in scope.
    a' x = [4mb[0m

Definitions:
  Definition 1:
    [4ma[0m x = b
  Definition 2:
    a [4mx[0m = b
  Definition 3:
      where [4mb[0m = b
  Definition 4:
    [4ma'[0m x = b
  Definition 5:
    a' [4mx[0m = b

Use sites:
  Definition used: 3
    a x = [4mb[0m
  Definition used: 3
      where b = [4mb[0m
